indico/modules/rb/client/js/common/linking/LinkBar.jsx

```diff
@@ -80,5 +93,6 @@ export default connect(
   }),
   dispatch => ({
     clear: bindActionCreators(linkingActions.clearObject, dispatch),
+    showNonOverlapping: bindActionCreators(linkingActions.showNonOverlapping, dispatch),
```

Comment:
I'd call this `toggleNonOverlapping` (both the prop and the action) since with the current name I'd expect it to be a bool with the current state...

Reply:
Done in [86e99da](https://github.com/indico/indico/pull/6846/commits/86e99da591b403f8706dbca2b4f7b09f04ed417b).